pybossa/util.py

```diff
@@ -908,23 +908,22 @@ def can_update_user_info(current_user, user_to_update):
     return False, None, None
 
 
-def get_enabled_users(user_emails):
+def mail_with_enabled_users(message):
     from pybossa.core import user_repo
```

Comment:
Minor: Can we put the import to the top of the file? I know sometimes the unit test would fail.

Reply:
this is required for tests to have app context.